### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by Maltergate** [PasaOpasen/geneticalgorithm2-6.8.7#20] [Q&A]
Hi,

Nothing to say, other than: thank you very much for this awesome package. It's really impressive, and so useful! I personally use it to tune Kalman filters with my simulations, and it works really great once we define the proper objective function.

Best

**Comment by PasaOpasen** [comment]
@Maltergate 
my pleasure

offer new ideas!

**Reply by Maltergate** [reply]
A friend of mine also advised me to take a look at the PyGMO library from the European Space Agency. [https://github.com/esa/pygmo2](https://github.com/esa/pygmo2). Do you know about it? It looks like they'd benefit from your algorithm!

**Reply by PasaOpasen** [reply]
@Maltergate interesting but heavy to install and too high-level, works not so fast
